src/main/java/org/jabref/gui/mergeentries/FetchAndMergeEntry.java

```diff
@@ -28,22 +21,18 @@
 import org.jabref.model.database.BibDatabaseContext;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.field.Field;
-import org.jabref.model.entry.field.FieldFactory;
 import org.jabref.model.entry.field.StandardField;
-import org.jabref.model.entry.types.EntryType;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/**
- * Class for fetching and merging bibliographic information
- */
 public class FetchAndMergeEntry {
 
-    // All identifiers listed here should also appear at {@link org.jabref.logic.importer.CompositeIdFetcher#performSearchById}
```

Comment:
Please keep my comment!

It is OK if AI deletes it. But then, re-add manually.